spec/integration_spec.rb

```diff
@@ -1,25 +1,19 @@
 
 context "Apivore tests running against a mock API" do
-  describe "valid swagger and correctly implemented API" do
-    it 'passes when the swagger description does match the implemented API' do
-      stdout = `rspec spec/data/example_specs.rb --example 'passing doc'`
-      expect(stdout).to match(/0 failures/)
-    end
-  end
 
-  describe "the swagger documents an unimplemented path" do
-    it 'should show an alert that the specific path is unimplemented' do
-      stdout = `rspec spec/data/example_specs.rb --example 'unimplemented path in doc'`
+  describe "the test checks undocumented path" do
+    it 'should show an alert that the specific path is undocumented' do
+      stdout = `rspec spec/data/example_specs.rb --example 'undocumented path in test'`
       expect(stdout).to match(/1 failure/)
-      expect(stdout).to match(/expected 200 array, got 404/)
+      expect(stdout).to match(/does not have a documented path/)
```

Comment:
I have fixed this. It does not say `200 array` anymore. I dont think it conveys what is intended. If the status codes do not match we should not check anything else